src/Gui/Forms/InitialPageInteractor.cs

```diff
@@ -182,5 +192,78 @@ private bool NeedsScanning()
                 Decompiler.Project != null &&
                 Decompiler.Project.Programs.Any(p => p.NeedsScanning);
         }
+
+        private void ProcessResources(ProgramResourceGroup prg, string outputDir)
+        {
+            try
+            {
+                if (prg.Name == "PE resources")
+                {
+                    // Create the dir
+                    Directory.CreateDirectory(outputDir + "\\resources");
+                    foreach (ProgramResourceGroup pr in prg.Resources)
+                    {
+                        switch (pr.Name)
+                        {
+                            case "CURSOR":
+                            {
+                                WriteResourceFile(outputDir + "\\resources", "Cursor", ".cur", pr);
+                            }
+                            break;
+                            case "BITMAP":
+                            {
+                                WriteResourceFile(outputDir + "\\resources", "Bitmap", ".bmp", pr);
+                            }
+                            break;
+                            case "ICON":
+                            {
+                                WriteResourceFile(outputDir + "\\resources", "Icon", ".ico", pr);
+                            }
+                            break;
+                            case "FONT":
+                            {
+                                WriteResourceFile(outputDir + "\\resources", "Font", ".bin", pr);
+                            }
+                            break;
+                            case "NEWBITMAP":
+                            {
+                                WriteResourceFile(outputDir + "\\resources", "NewBitmap", ".bmp", pr);
```

Comment:
Platform-specific code

Reply:
I like the initiative of extracting the resources of the binary as well. We will have to look into what would be needed to do the same for MacOS classic files.

This is PE-specific code, which should probably be handled by code in `PEImageLoader`. Since a proper abstraction is not yet in place for the manipulation embedded resources, let's not worry about it now. Please add a `//$TODO: move into PEImageLoader` comment so this doesn't get forgotten.